src/components/authentication/AdminDashboard.js

```diff
@@ -22,12 +23,16 @@ class UserIsAdmin extends Component {
         <Navbar {...this.props} />
         <Switch>
           <Route exact path="/">
+            <LandingPage />
+          </Route>
+          <Route exact path="/signin">
             <ImageManagement />
           </Route>
-          <Route path="/edit/:id" component={EditImage} />

```

Comment:
Line 28-30 is what I meant by cheeky way. Somehow after people log into their account (no matter they are admin or users), the route will automatically attach 'signin' to the end. 

The thing I wanted to achieve is here that once an admin signs in, the first thing they will see is the apartments/image page. Will ask you how to achieve it in a better way tomorrow.